ert3/engine/_run.py

```diff
@@ -18,6 +19,7 @@ def _prepare_experiment(
         workspace=workspace_root,
         experiment_name=experiment_name,
         parameters=parameter_names,
+        ensemble_size=ensemble_size,
```

Comment:
`EnsembleConfig` seems to have a `.size`. Why are we passing in `ensemble_size` separately?

Reply:
Would that be possible now @markusdregi after we moved the `_prepare_experiment` into the two different functions? Is the `EnsembleConfig` object updated with correct `ensemble_size` at the point where we lauch in `_prepare_sensitivity`?

Reply:
Because `EnsembleConfig` represents the user configuration and the user does not always get to decide the size of the ensemble. In the current implementation that is true when running sensitivity analysis.

We could of course make the configuration mutable and change it after the algorithm has ran, but I think that would be very unwise.